### DISCUSSION TITLE
Conquering cities

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1452] [Question and answer]
**Hello**,
I have a question regarding H&M cities: is it possible to conquer them? If yes, what are the benefits of doing that? 
_Thanks in advance for the answer!_

**Comment by Vdauphin** [comment]
Hello,
> is it possible to conquer them?

Yes

> what are the benefits of doing that?

No enemy patrol will spawn while intering inside a free city. Also this free city will not be use to spawn enemy patrol

Cheers

**Reply by AshTrayBY** [reply]
Are there any notifications or other visible confirmations that city is free?

Can it be recaptured by insurgents?

**Reply by Vdauphin** [reply]
> Are there any notifications or other visible confirmations that city is free?

In early game, the goal is not to free city as hideout will send reinforcement to recapture them. You can still talk to civilian to know how far are insurgents
After destroying all hideout, you will see red circle turning in green when liberating cities

**Reply by AshTrayBY** [reply]
Thanks for reply. Do you mean circles visible in debug mode?

**Reply by Vdauphin** [reply]
No, during the second phase, you will see red circle turning in green when liberating cities

**Reply by AshTrayBY** [reply]
Great. It seems we just played not enough time to see it cos restarted mission after every edit added. Thank you.

**Comment by yuanxiaoyu** [comment]
已经收到您的来信，我会尽快回复。同时祝您身体健康，工作愉快！袁晓渝致敬。
